Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -241,6 +241,7 @@ public class SchemeGenerator {
             macroExpansion: shouldExecuteOnLaunch ? nil : buildableReference,
             selectedDebuggerIdentifier: (scheme.run?.debugEnabled ?? Scheme.Run.debugEnabledDefault) ? XCScheme.defaultDebugger : "",
             selectedLauncherIdentifier: (scheme.run?.debugEnabled ?? Scheme.Run.debugEnabledDefault) ? XCScheme.defaultLauncher : "Xcode.IDEFoundation.Launcher.PosixSpawn",
+            askForAppToLaunch: scheme.run?.askForAppToLaunch,
```

Comment:
You would default here, using information from the rest of the target. If you have trouble with that, we can leave the existing behavior as the default for now.